<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>979</width>
    <height>863</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Виборчі округи</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>701</width>
      <height>211</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>150</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>№</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Виборчий округ</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Витрати на навчання</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Кількість спостерігачів</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Кількість членів ДВК</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string> Вит. на ДВК/спостерігача</string>
     </property>
    </column>
   </widget>
   <widget class="QComboBox" name="constituency">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>290</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>ТВО №117</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ТВО №118</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ТВО №119</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ТВО №120</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ТВО №121</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ТВО №122</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ТВО №123</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ТВО №124</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ТВО №125</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ТВО №126</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ТВО №127</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ТВО №128</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="add">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>290</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color:rgb(255, 255, 255);
background-color:rgb(206, 71, 69);
border: 2px solid black;
border-radius: 5px;
font: 14pt &quot;American Typewriter&quot;;
}
QPushButton:pressed 
{
    background-color: rgb(183, 62, 62);
}</string>
    </property>
    <property name="text">
     <string>Додати запис</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
color:rgb(255, 255, 255);
background-color:rgb(106, 195, 255);
border: 2px solid black;
border-radius: 5px;
font: 12pt &quot;American Typewriter&quot;;
}</string>
    </property>
    <property name="text">
     <string>Виборчий округ</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>250</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
color:rgb(255, 255, 255);
background-color:rgb(106, 195, 255);
border: 2px solid black;
border-radius: 5px;
font: 12pt &quot;American Typewriter&quot;;
}</string>
    </property>
    <property name="text">
     <string>Витрати на навчання</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>250</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
color:rgb(255, 255, 255);
background-color:rgb(106, 195, 255);
border: 2px solid black;
border-radius: 5px;
font: 12pt &quot;American Typewriter&quot;;
}</string>
    </property>
    <property name="text">
     <string>Кількість членів</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>250</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
color:rgb(255, 255, 255);
background-color:rgb(106, 195, 255);
border: 2px solid black;
border-radius: 5px;
font: 12pt &quot;American Typewriter&quot;;
}</string>
    </property>
    <property name="text">
     <string>Кількість спостерігачів</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="members">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>293</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>13</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>17</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>18</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="observants">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>290</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="education">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>290</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1000</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="singleStep">
     <number>500</number>
    </property>
   </widget>
   <widget class="QPushButton" name="sort">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>360</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color:rgb(255, 255, 255);
background-color:rgb(206, 71, 69);
border: 2px solid black;
border-radius: 5px;
font: 14pt &quot;American Typewriter&quot;;
}
QPushButton:pressed 
{
    background-color: rgb(183, 62, 62);
}</string>
    </property>
    <property name="text">
     <string>Сортувати за ДВК</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="avg">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>400</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>360</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
color:rgb(255, 255, 255);
background-color:rgb(106, 195, 255);
border: 2px solid black;
border-radius: 5px;
font: 12pt &quot;American Typewriter&quot;;
}</string>
    </property>
    <property name="text">
     <string>Сер. витрати/людина</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>550</y>
      <width>701</width>
      <height>181</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>150</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>№</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Виборчий округ</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Витрати на навчання</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Кількість спостерігачів</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Кількість членів ДВК</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="lessThanHalf">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>360</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color:rgb(255, 255, 255);
background-color:rgb(206, 71, 69);
border: 2px solid black;
border-radius: 5px;
font: 14pt &quot;American Typewriter&quot;;
}
QPushButton:pressed 
{
    background-color: rgb(183, 62, 62);
}</string>
    </property>
    <property name="text">
     <string>C &lt; 50%ДВК</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>360</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
color:rgb(255, 255, 255);
background-color:rgb(106, 195, 255);
border: 2px solid black;
border-radius: 5px;
font: 12pt &quot;American Typewriter&quot;;
}</string>
    </property>
    <property name="text">
     <string>Cер. ДВК витрати</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="avgDVK">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>400</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="memLessThanAvg">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>420</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color:rgb(255, 255, 255);
background-color:rgb(206, 71, 69);
border: 2px solid black;
border-radius: 5px;
font: 14pt &quot;American Typewriter&quot;;
}
QPushButton:pressed 
{
    background-color: rgb(183, 62, 62);
}</string>
    </property>
    <property name="text">
     <string>ДВК &lt; СЕР ДВК</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fiveByCondition">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>420</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color:rgb(255, 255, 255);
background-color:rgb(206, 71, 69);
border: 2px solid black;
border-radius: 5px;
font: 14pt &quot;American Typewriter&quot;;
}
QPushButton:pressed 
{
    background-color: rgb(183, 62, 62);
}</string>
    </property>
    <property name="text">
     <string>5 за критерієм</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="avgObsvCount">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>490</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>450</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
color:rgb(255, 255, 255);
background-color:rgb(106, 195, 255);
border: 2px solid black;
border-radius: 5px;
font: 12pt &quot;American Typewriter&quot;;
}</string>
    </property>
    <property name="text">
     <string>Cер. витрати спостерігача</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="avgObsv">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>490</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>450</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
color:rgb(255, 255, 255);
background-color:rgb(106, 195, 255);
border: 2px solid black;
border-radius: 5px;
font: 12pt &quot;American Typewriter&quot;;
}</string>
    </property>
    <property name="text">
     <string>Сер. к-сть спостерігачів</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="readFromFile">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>480</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color:rgb(255, 255, 255);
background-color:rgb(206, 71, 69);
border: 2px solid black;
border-radius: 5px;
font: 14pt &quot;American Typewriter&quot;;
}
QPushButton:pressed 
{
    background-color: rgb(183, 62, 62);
}</string>
    </property>
    <property name="text">
     <string>Зчитати з файлу</string>
    </property>
   </widget>
   <widget class="QPushButton" name="writeToFile">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>480</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color:rgb(255, 255, 255);
background-color:rgb(206, 71, 69);
border: 2px solid black;
border-radius: 5px;
font: 14pt &quot;American Typewriter&quot;;
}
QPushButton:pressed 
{
    background-color: rgb(183, 62, 62);
}</string>
    </property>
    <property name="text">
     <string>Записати у файл</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>240</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color:rgb(255, 255, 255);
background-color:rgb(206, 71, 69);
border: 2px solid black;
border-radius: 5px;
font: 14pt &quot;American Typewriter&quot;;
}
QPushButton:pressed 
{
    background-color: rgb(183, 62, 62);
}</string>
    </property>
    <property name="text">
     <string>Очистити </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>979</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
